checks/utils/windowsversions.py

```diff
@@ -40,7 +41,9 @@
 #
 # We probably don't need all the info here, but it comes in handy
 #
-# Ref: https://docs.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro
+# Ref: https://docs.microsoft.com/lifecycle/products/windows-10-home-and-pro
+#      https://aka.ms/WindowsUpdateHistory
+#      https://docs.microsoft.com/windows/release-health/release-information
```

Comment:
If updating the docs, use the learn.microsoft.com domain instead (which is what these redirect to).